src/components/SecondaryNav/SecondaryNav.story.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import SecondaryNav from './SecondaryNav';
+import SecondaryNavLink from './SecondaryNavLink';
+
+const stories = storiesOf('SecondaryNav', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add('SecondaryNav', () => (
+  <SecondaryNav
```

Comment:
wrap with a div that has `percy-min-width` class, this will keep the nav from collapsing it's width in regression tests